nautobot/project-static/js/forms.js

```diff
@@ -124,6 +124,8 @@ $(document).ready(function() {
     // API backed selection
     // Includes live search and chained fields
     // The `multiple` setting may be controlled via a data-* attribute
+    let api_version;
```

Comment:
Can we scope this variable more locally than this? In particular I'm wondering if this will have problems in the case where we might have multiple `APISelect` widgets with different `api_version` settings.

Now that we support `?api_version=X.Y.Z` as an alternative approach to the Accept header, maybe we could set the parameter directly in the `url` or `data` functions below instead?